src/docs/release.md

```diff
@@ -202,13 +202,11 @@ Name              | Email                       | PYPI Usename
 ------------------|-----------------------------|---------------
 John Sirois       | john.sirois@gmail.com       | john.sirois
 Benjy Weinberger  | benjyw@gmail.com            | benjyw
-Eric Ayers        | ericzundel@squareup.com     | ericzundel
```

Comment:
Although these removals reflect current pypi roles, I don't recall an emeritus or withdrawal dance for the removed users. Seems like we should have had one per the 'Committer Emeritus' section in https://www.pantsbuild.org/committers.html

Reply:
Hm... by "dance", do you mean the removal of commit access? Yea. Both that and removal of PyPI ownership seems to have been applied spottily. I actually removed the PyPI ownership while making these edits to match what was already in the `COMMITTERS` file.